### PR TITLE
Run tests in Docker

### PR DESCRIPTION
**Pull request by astencel-sumo**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1 @@
+Gemfile.lock
```
Comment (sumo-drosiek) [inline .dockerignore:1]: Shouldn't we base on the lock file?

File: DEVELOPER.md [hunk 2]
```diff
@@ -36,4 +45,4 @@ export sumo_url=https://events.sumologic.net/receiver/v1/http/XXXXXXXXXX
 rspec spec/
 ```
 
-The project is integrated to the Travis CI now. Make sure [all test passed](https://travis-ci.org/SumoLogic/logstash-output-sumologic) before creating PR
\ No newline at end of file
+The project is integrated to the Travis CI now. Make sure [all test passed](https://travis-ci.org/SumoLogic/logstash-output-sumologic) before creating PR
```
Comment (pmalek-sumo) [inline DEVELOPER.md:48]:
```suggestion
The project is integrated to Travis CI now.
Make sure [all test passed](https://travis-ci.com/SumoLogic/logstash-output-sumologic) before creating PR
```

Comment (astencel-sumo) [reply DEVELOPER.md]: Thanks! 🎉

File: .dockerignore (hunk 1 above)
Comment (astencel-sumo) [reply .dockerignore:1]: `Gemfile.lock` is not in the repo and it should not be, as for gems, you don't want to require specific versions of dependencies (or so I was told https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore). I added it to `.dockerignore` to make sure the locally generated file does not get into the Docker build.

Happy to discuss this.

Comment (astencel-sumo) [reply .dockerignore:1]: OK this kept bugging me, so I read a bit deeper and it seems reasonable to check in `Gemfile.lock` file after all, even though this is a gem.

See this comment https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore#comment75432623_4151540 and this answer https://stackoverflow.com/a/38964304/4603021 on the same question. I tend to agree with these.

If that's OK, I'll create a separate PR to add the `Gemfile.lock` to the repo and remove it from `.dockerignore`.

Comment (sumo-drosiek) [reply .dockerignore:1]: Thats fine 👍

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**